src/rnnodeapp/index.js

```diff
@@ -28,10 +28,16 @@ const styles = require('mapeo-styles');
 
 console.log('1: init')
 
-const USER_PATH = path.join(os.homedir(), 'mapeo', 'default');
+// NOTE: in the future, we might want separate private keys
+// for each user, so let's just have a 'default' user for now so it'll be
+// easier to migrate later
+const USER_ID = 'default' 
+const USER_PATH = path.join(os.homedir(), 'mapeo', USER_ID);
 const DB_PATH = path.join(USER_PATH, 'db');
 const MEDIA_PATH = path.join(USER_PATH, 'media');
-const STATIC_PATH = path.join(USER_PATH, 'static');
+
+const STATIC_PATH = path.join('/sdcard/Android/data/com.mapeomobile/static');
```

Comment:
@staltz is this consistent across android devices?

Reply:
Oh I don't believe it is, some devices might not even have an sdcard. The right way of doing this is fetching the path using Java APIs meant for this, passing the path string to the (frontend) JS, then passing that here (the backend). The package `react-native-fs` should help for doing the first part.

Reply:
ping @gmaclennan